### PR TITLE
pie chart updates

### PR DESCRIPTION
**Pull request by janwright73**
first pass at pie chart updates

### CODE REVIEW

File: react-charts/pie-chart/intro.md [hunk 1]
```diff
@@ -1 +1,5 @@
-In this scenario, you will learn how to use PatternFly React pie chart components for the first time.
+PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly.
+
+PatternFly provides a collection of React-based components you can use to build patterns with consistent markup, styling, and behavior.
```
Comment (gheake2) [inline react-charts/pie-chart/intro.md]:
```suggestion
PatternFly provides a collection of React-based components that are used to build patterns with consistent markup, styling, and behavior.
```

File: react-charts/pie-chart/intro.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+
+PatternFly React charts are composed using separate components, so features are more easily customizable. This course covers PatternFly pie charts - starting with a simple chart, tooltips, a legend, and concluding by changing the theme color.
```
Comment (gheake2) [inline react-charts/pie-chart/intro.md]:
```suggestion
PatternFly React charts are composed using separate components, so features are more easily customizable. This course covers PatternFly pie charts starting with a simple chart, tooltips, a legend, and concluding by changing the theme color.
```

File: react-charts/pie-chart/step2.md [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
+      /&gt;
+    &lt;/div&gt;
+  );
 }
-</pre>
 
-- Because Victory renders responsive containers, the Chart's width and height props do not determine the width and height of the chart in number of pixels, but instead define an aspect ratio for the chart. 
-The exact number of pixels will depend on the size of the container the chart is rendered into. 
-Typically, the parent container is set to the same width in order to maintain the aspect ratio.
+export default App;
```
Comment (gheake2) [inline react-charts/pie-chart/step2.md]:
```suggestion
export default `App;`
```

Comment (christiemolloy) [reply react-charts/pie-chart/step2.md]: dont make this change, this is the file import

File: react-charts/pie-chart/step4.md [hunk 4]
```diff
@@ -1,29 +1,48 @@
...
+  right: 140,
+  top: 20
+}}
 </pre>
 
-- The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered SVG
 
-4) Once the preview reloads - it should look like this:
-<img src="pie-chart/assets/tooltips.png" alt="Chart with tooltips" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+4) <strong>Add the `LegendOrientation` and `legendPosition` properties to the component.</strong>
```
Comment (christiemolloy) [inline react-charts/pie-chart/step4.md]: Split up the `legendOrientation` and `legendPosition` properties into individual steps. And add a step to add the `legendData`

**Pull request merged**